examples/virtual-device-2-service-token-connection/main.tf

```diff
@@ -0,0 +1,22 @@
+provider "equinix" {
+  client_id     = var.equinix_client_id
+  client_secret = var.equinix_client_secret
+}
+
+module "create_virtual_device_2_service_token_connection" {
```

Comment:
Added in slack, but adding here as well. What's the reason that we have this example separate from the service token creation example? Wouldn't it be useful for the customer to have it all together?

Reply:
We have also considered buyer seller scenario here,
Seller will create the service token and buyer will create separate connection using the same service token.